oraclizeAPI_0.4.sol

```diff
@@ -763,7 +763,7 @@ contract usingOraclize {
     }
 
     function oraclize_newRandomDSQuery(uint _delay, uint _nbytes, uint _customGasLimit) internal returns (bytes32){
-        if ((_nbytes == 0)||(_nbytes > 32)) throw;
+        require((_nbytes != 0) && (_nbytes <= 32));
```

Comment:
Agreed, more for clarity than for a security need. Since _nbytes is uint, sending a negative value would simply overflow the variable and that would be catched by the second conditions.